Documentation/QRCodeSign.md

```diff
@@ -0,0 +1,28 @@
+**Instructions**
+
+ All flowers are labeled, allowing visitors to take note of top performing cultivars under regional conditions.
+ 
+ <br>
+ 
+**Search for your favorite flower**
+
+ There are three ways to get to the flower you are looking for:
```

Comment:
Are we clarifying there are two ways to get into the system? Then it will be fine to say that. Otherwise it's confusing.